pkg/apis/maestro/v1alpha1/frameworkversion_types.go

```diff
@@ -88,6 +88,10 @@ type Parameter struct {
 	//Default is a default value if no paramter is provided by the instance
 	Default string `json:"default,omitempty"`
 
+	//Trigger identifies the plan that gets executed when this parameter changes in the Instance object.
+	//Default is `update` if present, or `deploy` if not present
+	Trigger string `json:"trigger,omitempty"`
```

Comment:
More general but, should we provide an (implementation independent) specification or at least part in the documentation?

Reply:
Can you expand on what you mean here?

Reply:
Some code independent specification/documentation: 
e.g., https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#container-v1-core